Source/SceneEditor/EditorCamera.cpp

```diff
@@ -141,50 +145,95 @@ namespace PlatinumEngine
 
 	void EditorCamera::MoveCamera(PlatinumEngine::Maths::Vec3 eulerAngle, PlatinumEngine::Maths::Vec3 translationValue)
 	{
+
+
 		// update euler angle
 		_eulerAngle += eulerAngle;
 
-		PlatinumEngine::Maths::Mat4 rotationMat4, translationMat4, translation2Mat4;
-
-		rotationMat4.SetRotationMatrix(-_eulerAngle);
+		PlatinumEngine::Maths::Mat4 rotationMat4, translationMat4, rotationMat4Inverse, translationMat4Inverse;
 
+		// get rotation matrix
+		rotationMat4Inverse.SetRotationMatrix(-_eulerAngle);
```

Comment:
mathematics muhahaha

Reply:
muhahaha, this will be changed in gizmo branch.
do i need to do it here?